Tools/Bluetooth/BLE_hci.py

```diff
@@ -295,19 +295,22 @@ def wait_events(self, seconds = 2, print_evt = True):
      # Send a HCI command to the serial port. Will add a small delay and wait for
      # and print an HCI event by default.
     ################################################################################
-    def send_command(self, packet, resp = True, delay = 0.01, print_cmd = True, timeout=16):
+
+    def send_command(self, packet, resp = True, delay = 0.01, print_cmd = True, timeout=6):
         # Send the command and data
         if(print_cmd):
             if self.id == "-":
                 print(str(datetime.datetime.now()) + "  >", packet)
             else:
                 print(str(datetime.datetime.now()) + f" {self.id}>", packet)
 
-        print('Packet', packet)
+
```

Comment:
Can we just remove these lines? I think we already have the option to print the packets.

Reply:
Yes i will remove it